dspace-server-webapp/src/main/java/org/dspace/app/rest/submit/step/validation/MetadataValidation.java

```diff
@@ -66,13 +68,43 @@ public List<ErrorRest> validate(SubmissionService submissionService, InProgressS
         if (documentType.size() > 0) {
             documentTypeValue = documentType.get(0).getValue();
         }
+        Map<String, Boolean> isAllowedLookup = new HashMap<>();
```

Comment:
Yes, I can try to normalise this.
The Map was put in place in case I wanted to have true/false values associated for "required" config, but I think I avoided the need for that, so I can simplify to a List.

Reply:
@tdonohue should be good to go now thanks!